README.md

```diff
@@ -106,6 +106,17 @@ The prover node requires a configuration file to be passed in at runtime.
 - `server_url` - The URL of the server to connect to for tasks. The provided URL is the dockers reference to the host machines 'localhost'
 - `priv_key` - The private key of the prover node. This is used to sign the tasks and prove the work was done by the prover node.
 
+### Dry Run Service Configuration
+
+The Dry Run service will be required to run parallel to the prover node. The Dry Run service is responsible for synchronising tasks with the server and ensuring the prover node is working correctly.
+This service must be run in parallel to the prover node, so running the service through docker compose is recommended.
+
+In the `dry_run_config.json` file, modify the connection strings to the server and the MongoDB instance.
+
+- `server_url` - The URL of the server to connect to for tasks. Ensure this is the same as the prover node.
+- `mongodb_uri` - The URI of the MongoDB instance to connect to.
+- `priv_key` - Private Key is NOT used in the Dry Run service when run parallel to the prover node, so it can be ignored.
```

Comment:
We need discuss this one as we have outer docker so normally rest server need filter the prover dry run service request too.

Reply:
After discuss we will leave it like this as prover dry run just query, do not change anything